### PR TITLE
Unittest on check_partials in subsystems on gasp_based side

### PR DESCRIPTION
**Pull request by xjjiang**
### Summary

This PR adds all missing check_partials unit tests in:
- subsystems/aerodynamics/gasp_based/test
- subsystems/energy/test
- subsystems/geometry/gasp_based/test
- subsystems/mass/gasp_based/test
- subsystems/propulsion/test

**Note:** The check_partials in `test_electroprop()` of `TurbopropTest` is commented out. Otherwise, `inf` is generated.

### Related Issues

- Resolves #

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

**Review by Kenneth-T-Moore — APPROVED** [review]

File: aviary/subsystems/geometry/gasp_based/test/test_wing.py [hunk 1]
```diff
@@ -331,6 +331,10 @@ def test_case1(self):
 
 
 class WingGroupTestCase2(unittest.TestCase):
+    """
+    Wing with both folds and struts which has fold dimensional location and strut dimensional location specified
```
Comment (crecine) [inline aviary/subsystems/geometry/gasp_based/test/test_wing.py:335]: add `with the fold at the strut connection`

Comment (crecine) [reply aviary/subsystems/geometry/gasp_based/test/test_wing.py:335]: to differentiate it from case4

File: aviary/subsystems/aerodynamics/gasp_based/test/test_common.py [hunk 2]
```diff
@@ -4,9 +4,34 @@
 import openmdao.api as om
 from openmdao.utils.assert_utils import assert_check_partials, assert_near_equal
 
-from aviary.subsystems.aerodynamics.gasp_based.common import (CLFromLift,
+from aviary.subsystems.aerodynamics.gasp_based.common import (AeroForces, CLFromLift,
```
Comment (crecine) [inline aviary/subsystems/aerodynamics/gasp_based/test/test_common.py:7]: I'm not sure if there are the right number of spaces, but split this so there is one per line, like the other two imported from here.
```suggestion
from aviary.subsystems.aerodynamics.gasp_based.common import (AeroForces,
                                                                                                             CLFromLift,
```

**Review by crecine — APPROVED** [review]

File: aviary/subsystems/geometry/gasp_based/test/test_wing.py (hunk 1 above)
Comment (xjjiang) [reply aviary/subsystems/geometry/gasp_based/test/test_wing.py:335]: done

File: aviary/subsystems/aerodynamics/gasp_based/test/test_common.py (hunk 2 above)
Comment (xjjiang) [reply aviary/subsystems/aerodynamics/gasp_based/test/test_common.py:7]: Done.

**Pull request merged**